app/models/manageiq/providers/infra_manager.rb

```diff
@@ -44,5 +44,24 @@ def self.merge_config_settings(cfg = VMDB::Config.new("vmdb"))
       path = [:ems, :ems_redhat, :service, :read_timeout]
       cfg.merge_from_template_if_missing(*path)
     end
+
+    def self.credential_validation(ems, auth_type, creds, revalidate)
+      ems.update_authentication(creds, :save => false)
```

Comment:
`ems.authentication_check` uses `:save => revalidate` which I did keep that way, `ems.update_authentication` uses `:save=> (mode != :validate)` and I figured we're always validating in validate_credentials but that could be completely wrong.